test/chat_api/conversations_test.exs

```diff
@@ -286,6 +286,20 @@ defmodule ChatApi.ConversationsTest do
       refute conv2.archived_at
     end
 
+    test "update_conversation/2 sets the closed_at field based on updated status", %{
```

Comment:
hooray for tests 🎉 can we add one to `conversation_controller_text.exs` as well?

Reply:
Can I ask how should I perform the controller test for these changes? Do I need to add `closed_at` field in "basic.json" in `ChatApiWeb.ConversationView`? I'm not sure about this one.

Hope for your guidance on this one so I could perform the remaining tests.
Thank you.

Reply:
yup you can add `closed_at` to both `basic.json` and `expanded.json`, and then make sure this value is changed correctly when the `status` field is updated

Reply:
Would it be ok to just assert the `closed_at` field like  `assert closed_at != nil` if status is "closed" while ` assert closed_at == nil` if status is "open" ?

Reply:
yup that's fine 👍